Ntust_Cloud_System/Ntust_Cloud_System/FileOperator/FileOperator.h

```diff
@@ -0,0 +1,30 @@
+//
+//  FileOperator.h
+//  Ntust_Cloud_System
+//
+//  Created by 林仁鴻 on 2021/4/13.
+//
+
+
+#ifndef FileOperator_h
```

Comment:
Define guard should follow coding style guide's syntax, please see:
https://google.github.io/styleguide/cppguide.html#The__define_Guard